src/Microsoft.ML.LightGbm/LightGbmArguments.cs

```diff
@@ -58,6 +59,11 @@ public BoosterParameterBase(OptionsBase options)
         public abstract class OptionsBase : IBoosterParameterFactory
         {
             internal BoosterParameterBase GetBooster() { return null; }
+            /// <summary>
+            /// Whether training data is unbalanced. Used by <see cref="LightGbmBinaryTrainer"/>.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Use for binary classification when training data is not balanced.", ShortName = "us")]
```

Comment:
> Use for binary classification when training data is not balanced.

Is putting this option on the `OptionsBase` class correct if it is only for binary classification?

Reply:
You're right. The cause of the problem was this [commit](https://github.com/dotnet/machinelearning/pull/2948/files#diff-8173688cd56a37cbcc92012f7d22a31dL152) which split the trainers in to their classes (binary and multiclass). is it better not to include it as an argument in this file?